steps/browserstack-bitrise-espresso-step/1.0.0/step.yml

```diff
@@ -0,0 +1,252 @@
+    - "false"
+- opts:
+    category: Test Configuration
+    description: The single runner invocation feature runs each test case in a single
+      process that reduces the overall build time. [Learn more.](https://www.browserstack.com/docs/app-automate/espresso/single-runner-invocation)
+    summary: Enable to run each test case in a single process to reduce overall build
+      time.
+    title: Single runner invocation
+    value_options:
+    - "true"
+    - "false"
+  use_single_runner_invocation: "false"
+- opts:
+    category: Test Configuration
+    description: A callback URL helps you enable BrowserStack notify you upon the
+      completion of all builds under a given project. You will receive a callback
+      if no new builds triggered for 5 minutes and all existing builds have completed
+      their execution.This parameter requires that the Project name parameter also
+      be set.
+    summary: 'A callback URL to enable BrowserStack notify about completion of build
+      under a given project. '
+    title: Notify project status
+  project_notify_url: null
+- opts:
+    category: Test Configuration
+    description: Enable local testing to retrieve app data hosted on local/private
+      servers
+    summary: Enable local testing to retrieve app data hosted on local/private servers
+    title: Local testing
+    value_options:
+    - "true"
+    - "false"
+  use_local: "false"
+- opts:
+    category: Test Configuration
+    description: A mock web server mocks the behavior of an actual remote server.
+      This makes it easy to test different scenarios without depending on your remote
+      server and without having to make changes to your remote server. Local Testing,
+      Network logs, and IP geolocation will not work if you are using a mock server.
+      [Learn more.](https://www.browserstack.com/docs/app-automate/espresso/use-mock-server)
+    summary: Enable to mock a web server in your espresso tests to mock your API responses.
+      Learn more.
+    title: Mock server
+    value_options:
+    - "true"
+    - "false"
+  use_mock_server: "false"
+- api_params: null
+  opts:
+    category: Test Configuration
+    description: |
+      New line key-value pairs of capabilities provided by BrowserStack.
+
+      For example:
+      coverage=true
+
+      geoLocation=CN
+
+      [Learn about more capabilities.](https://www.browserstack.com/docs/app-automate/api-reference/espresso/builds#execute-a-build)
+    summary: New line key-value pairs of capabilities provided by BrowserStack.
+    title: Test capabilities
+- check_build_status: "true"
+  opts:
+    category: Test Configuration
+    description: Wait for build results
+    summary: Wait for BrowserStack to complete the execution and get the test results
+    title: Build Status
+    value_options:
+    - "true"
+    - "false"
+outputs:
+- BROWSERSTACK_BUILD_URL: null
+  opts:
+    description: BrowserStack Dashboard url for the executed build
+    summary: ""
```

Comment:
This should never be an empty string, either remove this or use the same sentence as in the description.